contrib/inventory/ec2.py

```diff
@@ -491,9 +491,14 @@ def get_rds_instances_by_region(self, region):
         try:
             conn = self.connect_to_aws(rds, region)
             if conn:
-                instances = conn.get_all_dbinstances()
-                for instance in instances:
-                    self.add_rds_instance(instance, region)
+                marker = None
+                while True:
+                    instances = conn.get_all_dbinstances(marker=marker)
+                    marker = instances.next_marker
```

Comment:
The other boto APIs use next_marker and the code docs themselves seem ambiguous so ... will fix to use marker in this case but I suspect it might change in the future as they align the APIs.